test/integration/patients/patient/dashboard.js

```diff
@@ -31,7 +40,7 @@ function createActionPostRoute(id) {
 context('patient dashboard page', function() {
   specify('action and flow list', function() {
     const testTime = dayjs(testDate()).hour(12).valueOf();
-    const actionData = {
+    const testActionData = getAction({
       id: '1',
```

Comment:
it would be nice to not set this and just use `testActionData.id` where `1` is currently needed.

Reply:
Updated 👍